### PR TITLE
Add camel-zookeeper wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-zookeeper wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -2157,21 +2157,19 @@
...
-<!--        <bundle dependency='true'>mvn:com.google.guava/guava/${zookeeper-guava-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-zookeeper/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-zookeeper' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jackson</feature>
+        <bundle dependency='true'>wrap:mvn:org.apache.zookeeper/zookeeper/${zookeeper-version}$Export-Package=org.apache.zookeeper;version=${zookeeper-version},org.apache.zookeeper.*;version=${zookeeper-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Can be simplified using the pattern `org.apache.zookeeper*`

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -2157,21 +2157,19 @@
...
+        <bundle dependency='true'>wrap:mvn:org.apache.zookeeper/zookeeper-jute/${zookeeper-version}$Export-Package=org.apache.jute;version=${zookeeper-version},org.apache.zookeeper.*;version=${zookeeper-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Can be simplified using the pattern `org.apache.*`

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: Ok done

**Review by essobedo — APPROVED** [review]
LGTM 👍 thx

**Pull request merged**